### DISCUSSION TITLE
What version of postgres does Jellystat use?

### DISCUSSION THREAD

**Discussion by ghost** [CyferShepard/Jellystat#64] [Q&A]
What version of postgres does Jellystat use?

The docker compose example doesn't specify what version. This will lead to something breaking at some point.

**Comment by TheMapledCog** [comment]
Hey @douglasparker, based on the docker-compose file provided the postgres image version is not specified, so it will pull the image with the [latest](https://hub.docker.com/layers/library/postgres/latest/images/sha256-2e89ed90224245851ea2b01e0b20c4b893e69141eb36e7a1cece7fb9e19f21f0?context=explore) tag.

I have done some tested running on postgres:14 and am currently testing running on postgres:15
